phc-rails/app/models/event_registration.rb

```diff
@@ -15,4 +15,25 @@
 class EventRegistration < ActiveRecord::Base
   has_and_belongs_to_many :services
 
+  def self.phc_event
+    # PHC 58 - PRODUCTION
+    "a0R40000007kSvU"
+  end
+
+  def to_salesforce_object
```

Comment:
Assignment Branch Condition size for to_salesforce_object is too high. [15.17/15]